includes/permissions.sh

```diff
@@ -0,0 +1,31 @@
+# shellcheck disable=SC2148
+
+# Changes the ownership to $APP_USER:$APP_GROUP for the given file / recursively to the given folder
+change_ownership() {
```

Comment:
I dont get the idea of this alias function/wrapper, why would you not use just chown -R or chown in the file directly, it feels massively overcomplicated to do a one-liner.

I dont get the idea and the thoughts behind this, please explain.

Reply:
See https://github.com/jammsen/docker-palworld-dedicated-server/pull/240#discussion_r1506894930